modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/upgrade/test/SXPBlueprintUpgradeProcessTest.java

```diff
@@ -398,9 +411,18 @@ private void _assertSXPElement(
 			sxpElementId);
 
 		Assert.assertNotNull(sxpElement);
-		Assert.assertEquals(
-			expectedElementDefinitionJSON,
-			sxpElement.getElementDefinitionJSON());
+
+		if (Validator.isBlank(expectedElementDefinitionJSON)) {
+			Assert.assertTrue(
```

Comment:
just to confirm, what is the case we can have it blank?

Reply:
shouldnt it be an empty json?

Reply:
the JSONAssert throws an exception if you pass an empty string.